app_integrations/apps/app_base.py

```diff
@@ -222,6 +222,24 @@ def _make_request(self, full_url, headers, params):
 
         return response.json()
 
+    def _make_post_request(self, full_url, headers, json):
+        """Method for returning the json loaded response for this POST request
+
+        Returns:
+            bool or dict: False if the was an error performing the request,
+                or a dictionary loaded from the json response
+        """
+        LOGGER.debug('Making request for service \'%s\' on poll #%d',
```

Comment:
Can you add `post` to this logger message somewhere, ie: `Making post request for...`